src/REPL/Repl.cpp

```diff
@@ -104,10 +133,19 @@ namespace Ark
     void Repl::trim_whitespace(std::string& line)
     {
         size_t string_begin = line.find_first_not_of(" \t");
-        if (std::string::npos != string_begin)
+        if(std::string::npos != string_begin)
         {
             size_t string_end = line.find_last_not_of(" \t");
             line = line.substr(string_begin, (string_end - string_begin + 1));
         }
     }
-}
\ No newline at end of file
+
+    void Repl::cgui_setup()
+    {
+        using namespace std::placeholders;
+
+        m_repl.set_completion_callback(std::bind(&hook_completion, _1, _2, std::cref(KeywordsDict)));
+        m_repl.set_highlighter_callback(std::bind(&hook_color, _1, _2, cref(ColorsRegexDict)));
```

Comment:
cref and not std::cref? it's not the same function?

Reply:
I forget to add `std`

Reply:
...